Changelog.md

```diff
@@ -1,3 +1,7 @@
+-----------Changelog for v1.1.18.1-----------
+
+- Fixed the "player detection mod compatibility" feature not working

```

Comment:
For the changelog, please change this line to "- Ported to 1.21.8", and delete everything below this line, so this is the only section in the changelog.

(I hope there's not another hotfix coming, lol)